### ISSUE TITLE
Labels partially transformed into numbers when --hex is on

### ISSUE THREAD

**Issue by dpt** [skoolkid/skoolkit#92]
I've got a chunk like this in my skool file with some partially numeric labels:
```
w$6B79 DEFW roomdef_3_hut2_right_bed_A
 $6B7B DEFW roomdef_3_hut2_right_bed_B
 $6B7D DEFW roomdef_3_hut2_right_bed_C
 $6B7F DEFW roomdef_3_hut2_right_bed_D
 $6B81 DEFW roomdef_3_hut2_right_bed_E
 $6B83 DEFW roomdef_3_hut2_right_bed_F
```

and if I do this:
`skool2asm.py --hex -c build/../TheGreatEscape.skool > build/TheGreatEscape.asm`

Then I get hex in the middle of the labels:
```
beds:
  DEFW roomdef_$0003_hut2_right_bed_A
  DEFW roomdef_$0003_hut2_right_bed_B
  DEFW roomdef_$0003_hut2_right_bed_C
  DEFW roomdef_$0003_hut2_right_bed_D
  DEFW roomdef_$0003_hut2_right_bed_E
  DEFW roomdef_$0003_hut2_right_bed_F
```

Which obviously screws up the assembly.

It doesn't happen without `--hex`.

**Comment by skoolkid** [comment]
Are you sure you want to use labels in these DEFW statements in the skool file? If you do, you'll lose the hyperlinks in the HTML disassembly.

Anyway, this has been fixed in 5fa7139a30e5b650cd93c7320fe3d60acaf795a8.

**Comment by dpt** [comment]
Gah, I hadn't spotted that!
